sdk/storage/azblob/appendblob/client.go

```diff
@@ -36,11 +36,13 @@ func NewClient(blobURL string, cred azcore.TokenCredential, options *ClientOptio
 	authPolicy := shared.NewStorageChallengePolicy(cred)
 	conOptions := shared.GetClientOptions(options)
 	conOptions.PerRetryPolicies = append(conOptions.PerRetryPolicies, authPolicy)
-	pl := runtime.NewPipeline(exported.ModuleName,
-		exported.ModuleVersion, runtime.PipelineOptions{},
-		&conOptions.ClientOptions)
 
-	return (*Client)(base.NewAppendBlobClient(blobURL, pl, nil)), nil
+	azClient, err := azcore.NewClient("appendblob.Client", exported.ModuleVersion, runtime.PipelineOptions{}, &conOptions.ClientOptions)
```

Comment:
Nit: you could consolidate all of the `"appendblob.Client"` strings into a `const` (comment applies to all clients).

Reply:
Done!